loaders/src/glTF/2.0/glTFLoaderExtension.ts

```diff
@@ -104,6 +104,15 @@ export interface IGLTFLoaderExtension extends IGLTFBaseLoaderExtension, IDisposa
      */
     loadAnimationAsync?(context: string, animation: IAnimation): Nullable<Promise<AnimationGroup>>;
 
+    /**
+     * Define this method to modify the default behavior when loading skins.
+     * @param context The context when loading the asset
+     * @param node The glTF node property
+     * @param skin The glTF skin property
+     * @returns A promise that resolves with the loaded Babylon skeleton when the load is complete or null if not handled
+     */
+    loadSkinAsync?(context: string, node: INode, skin: ISkin): Nullable<Promise<void>>;

```

Comment:
Let's make this an underscore function for now. The questions for this contract is whether we need an assign function like the some of the other functions and whether the skeleton should be returned in the promise. I think the answer is yes to both.

```suggestion
    _loadSkinAsync?(context: string, node: INode, skin: ISkin): Nullable<Promise<void>>;
```

Reply:
Changes Made :)